app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,84 @@
+class ReviewsController < ApplicationController
+  before_action :require_login, only: [:new, :edit, :create, :update, :destroy]
+
+  def new
+    @bar = Bar.find(params[:bar_id])
+    @review = Review.new
+  end
+
+  def create
+    @bar = Bar.find(params[:bar_id])
+    @review = Review.new(review_params)
+    @review.user = current_user
```

Comment:
you can set the current user when you call `Review.new` like this:

```ruby
Review.new(review_params, user: current_user, bar: @bar)
```

I think this would be a little cleaner, not a huge deal though. Keep in mind you also only need to make variables declared in the controller 'instance' variables (with the `@` symbol) if you need them to be defined in your views.